js/app.js

```diff
@@ -1 +1,121 @@
-console.log('Hello frontend');
+// Logic Reference: https://github.com/angle943/tic-tac-toe
+// have bugs when meet condition 2=5=8, 3=6=9, 4=5=6, 7=8=9
+// add if(one) etc. to check if it is empty ''
+
+// Variables
+
+msg = document.querySelector('#message h2')
+xScores = document.getElementById('xScores')
+oScores = document.querySelector('#oScores')
+tieScores = document.querySelector('#tieScores')
+let xWins = 0
+let oWins = 0
+let tie = 0
+let moves = 0
+let next = ''
+let gameIsGoing = true
+console.log(xScores, oScores);
+
+// Functions
+
+// display winner
+function display(player) {
+    if (player === 'X'){
+        msg.textContent = 'X Wins!!!'
+        gameIsGoing = !gameIsGoing
+        xWins ++
+        xScores.textContent = xWins
+    }
+    else if (player === 'O') {
+        msg.textContent = 'O Wins!!!'
+        gameIsGoing = !gameIsGoing
+        oWins ++
+        oScores.textContent = oWins
+    }
+    // some boxs have '', which also meet the if statement, took time to figure out...
+    // else {
+    //     msg.textContent = next + ' is next'
+    // }
+}
+
+// check winning cases
+function gameStatus(){
```

Comment:
Similar to what I mentioned about naming above.  A good rule of thumb for function names is to use verbs because functions DO something.  This function is checking the status of the game, so maybe "checkGameStatus" would be a good name.